example/example.documentation.yaml

```diff
@@ -163,7 +163,6 @@ paths:
                 value:
                   key: 1234-5678-90
                   name: John Doe
-                  birthday: 1963-04-21T00:00:00.000Z
```

Comment:
check why it's gone

Reply:
because pulling examples in mixer and that does not involve transformations.
while instead should probably do it in `flattenIO()` and then revert c1e111c612773703551d4ede65944d8596e62368

Reply:
addressed in #2651